covid.py

```diff
@@ -1 +1,67 @@
-#this is comment
+import argparse
+import json
+import requests
+
+
+def parser():
+    my_parser = argparse.ArgumentParser(
+        description="By giving name of country you will get "
+        "percentage of people recovered and still sick from all population"
+    )
+    my_parser.add_argument(
+        "--country",
+        action="store",
+        required=True,
+        help="Name of country",
+    )
+    return my_parser.parse_args()
+
+
+def computing_answer(population, confirmed_cases, recovered_people, dead_people):
+    percent_of_recovered = round(100 * recovered_people / population, 1)
+    percent_of_sick = round(
+        100 * (confirmed_cases - recovered_people - dead_people) / population, 1
+    )
+    print(
+        json.dumps(
+            {"recovered": percent_of_recovered, "sick": percent_of_sick}, indent=1
+        )
+    )
+
+
+def check_http_response():
+    response = requests.get(url="https://covid-api.mmediagroup.fr/v1/cases")
```

Comment:
You're close, but a bit wrong. You're calling an API, checking response, then calling API once [again](https://github.com/JanZSwier/Mentoring_task1/pull/2/files#diff-d79745fed2acc69ff19556390477a4f6ac7a2514479af57db1c14f76c3e21af3R43)!!

Ideally, it should look like this (pseudocode):
```
response = call_covid_api()
if response.ok:
    # let's do the computation
else:
    # print oh no, try again later message